.github/workflows/vulnerability-monitor.yaml

```diff
@@ -0,0 +1,37 @@
+name: Vulnerability-Monitor
+
+on:
+  schedule:
+    - cron: "0 16 * * *"
+
+jobs:
+  docker-image-monitor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - run: |
+          sudo apt-get update
+          sudo apt-get install jq
+          curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s --
+          docker scout cves twingate/kubernetes-operator:dev --only-fixed --format sarif --output kubernetes-operator.sarif.json
+          if [ `cat kubernetes-operator.sarif.json | jq -r '.runs[0].results | length'` -gt 0 ];
+          then docker scout cves twingate/kubernetes-operator:dev --only-fixed --output kubernetes-operator.txt;
```

Comment:
1. Code isn't properly indented
2. You dont need to run docker scout twice and use `jq` to check if there are only error you can check exit code:
```
  -e, --exit-code                   Return exit code '2' if vulnerabilities are detected
```
3. lets add `--only-severity` for medium\high\critical